yarn.lock

```diff
@@ -595,6 +595,15 @@
     "@solana/spl-token" "^0.1.8"
     decimal.js "^10.3.1"
 
+"@orca-so/common-sdk@^0.0.7":
```

Comment:
`@orca-so/orca-sdk` pulls in `@orca-so/common-sdk` as a dependency, should we also bump there?

I think there's also an extraneous `"@orca-so/common-sdk@^0.0.4":` above